entries/entry05.md

```diff
@@ -1,8 +1,100 @@
 # Entry 5
```

Comment:
Sources: 1
EDP: 1
Knowledge: 1
Skills: 1
Professionalism: 0 (I commented on your last blog entry on 3/13 to use `java` after your opening triple-backticks to get syntax highlighting.